LayoutTests/fast/dom/gc-abort-controller.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Test AbortController memory leak with event listeners</title>
+    <script src="../../resources/js-test.js"></script>
+</head>
+<body>
+<script>
+jsTestIsAsync = true;
```

Comment:
Please add a `description("Test AbortController memory leak with event listeners");` before this line and rebaseline the test.